cmd/app/app_test.go

```diff
@@ -40,42 +42,10 @@ func TestWorkspaceCommand(t *testing.T) {
 	listFunc = listPkgMock.List
 	listPkgMock.On("List").Return(nil)
 
-	err := cmd.Execute()
+	err := cmd.ExecuteContext(ctx)
 	if err != nil {
 		assert.Fail(t, "cmd.Execute had unexpected error")
 	}
 
 	listPkgMock.AssertCalled(t, "List")
 }
-
-// TODO: this test may need a stubbed out parent (root) command to get aliasing working
-/*
-func TestPostRunWorkspaceDeprecationMessage(t *testing.T) {
```

Comment:
note: 🪓 Don't think we need this commented out test. We can add a fresh test when want to tackle that test coverage.

Reply:
Ocuh! Commented codeblocks are often painful to revisit when the full context is in `git` histories 👾

Thanks for cleaning this up!

Reply:
Totally agree! The `git` histories will keep the chronicle in case we ever want to revisit it. 📜